cpp/include/rmm/cuda_device.hpp

```diff
@@ -16,18 +16,15 @@
 #pragma once
 
 #include <rmm/aligned.hpp>
-#include <rmm/detail/error.hpp>
 #include <rmm/detail/export.hpp>
 
-#include <cuda_runtime_api.h>
-
 #include <cstddef>
 #include <utility>
 
-namespace RMM_NAMESPACE {
+namespace RMM_EXPORT rmm {
 
 struct cuda_device_id;
-inline cuda_device_id get_current_cuda_device();
+RMM_EXPORT cuda_device_id get_current_cuda_device();
```

Comment:
These `RMM_EXPORT` on a per device function don't seem needed

Reply:
Fixed in a29a2f9b4b07c4b37ab9917443cfd8e52beefbb7.